swagger.yaml

```diff
@@ -1819,6 +1819,14 @@ components:
           type: integer
         status:
           $ref: '#/components/schemas/ProjectStatus'
+        estimatedStartDate:
+          description: Estimated start date of the project, cannot be changed after the project has started
+          format: date
+          type: string
+        estimatedEndDate:
+          description: Estimated end date of the project, cannot be changed after the project has started
```

Comment:
Hmm, as far as I understood from the design and the overal term "estimated", the start and end dates can be edited

Reply:
If I understood correctly, this is supposed to be an estimate that is set before the project starts, and in the future the customer wants to have some suggestion for the end time based on the set start time and the actual duration of the previously done projects of the same time. Hence, in my opinion it should probably be uneditable after the start of the project. 🤔